ios/RNZendeskChat.m

```diff
@@ -346,3 +350,18 @@ - (void) registerForNotifications:(NSData *)deviceToken {
    [ZDKChat registerPushToken:deviceToken];
 }
 @end
+
+@interface NavigationControllerWithCompletion ()
+
+@end
```

Comment:
You have already put the property in the public header, ain't it? I suggest to remove this part if not used or remove the public part of the interface and use only the private one. What do you think?